src/Quizzer.API.App/Controllers/AnswerController.cs

```diff
@@ -0,0 +1,79 @@
+using Microsoft.AspNetCore.Mvc;
+using Quizzer.API.BL.Facades;
+using Quizzer.API.BL.Models;
+
+namespace Quizzer.API.Controllers {
+    [ApiController]
+    [Route("api/answers")]
+    public class AnswerController : ControllerBase {
+        private readonly ILogger<AnswerController> _logger;
+        private readonly AnswerFacade _answerFacade;
+
+        public AnswerController(ILogger<AnswerController> logger,
+            AnswerFacade answerFacade) {
+            _logger = logger;
+            _answerFacade = answerFacade;
+        }
+
+        [HttpGet("", Name = "GetAll")]
+        public IEnumerable<AnswerDetailModel> GetAll() {
+            return _answerFacade.GetAll();
+        }
+
+        [HttpGet("{id:guid}", Name = "GetById")]
+        [ProducesResponseType(StatusCodes.Status404NotFound, Type = typeof(Guid))]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(AnswerDetailModel))]
+        public IActionResult GetById(Guid id) {
+            AnswerDetailModel answer = _answerFacade.GetDetailById(id);
+            
+            if (answer.Id != id)
+                return NotFound(id);
```

Comment:
It should leave default value Guid.Empty, altough this is not ideal. What if we return nullable from Facade to indicate it may fail and you need to check the result?

Reply:
Sounds good, should I do it before the end of the first phase?

Reply:
If you were to change it, I would say now, but I don't think it's high priority.